creme/settings.py

```diff
@@ -750,7 +750,10 @@
     ('creme.persons',          'persons/css/persons.css'),
 
     ('creme.activities',       'activities/css/activities.css'),
-    ('creme.activities',       'activities/css/fullcalendar-3.10.2.css'),
+    # ('creme.activities',       'activities/css/fullcalendar-3.10.2.css'),
+    # ('creme.activities',       'activities/css/fc3-creme.css'),
```

Comment:
Can you add a comment like "to be deleted in creme 2.7' please (in order in do not forget to remove the related files) ?